### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dsikes**

### CODE REVIEW

File: lockbox.go [hunk 1]
```diff
@@ -18,21 +15,21 @@ const (
...
+	Locked           bool
+	CurrentUser      *User
+	CurrentNamespace string
 }
 
 // GetLockbox will return an instance of lockbox
-func GetLockbox(name, master string) (*Lockbox, error) {
-	db, err := bolt.Open(fmt.Sprintf("%v.lockbox", name), 0600, nil)
+func GetLockbox(lbname string) (*Lockbox, error) {
+	db, err := bolt.Open(lbname, 0600, nil)
```
Comment (iAziz786) [inline lockbox.go:27]: Glad that it will be just the name in lieu of `*.lockbox`

File: lockbox.go [hunk 2]
```diff
@@ -134,76 +145,88 @@ func (l *Lockbox) Lock() error {
...
 	if err != nil {
 		return err
 	}
-	securePassphrase := pbkdf2.Key([]byte(l.MasterKey), []byte(salt), 4096, 10, sha1.New)
-	keyurl, err := decrypt(enckey, string(securePassphrase))
+
+	user := &User{
+		Username: username,
+		Password: password,
+	}
```
Comment (iAziz786) [inline lockbox.go:158]: Everything here seems fine, the only thing that I think we can do better is the creation of the user. We can have a construction function (`NewUser`) which will take the only username and password. By doing that way we can also set the `EncryptionKey` at the time of instantiating.

**Review by iAziz786 — COMMENTED** [review]
Hey @dsikes, I think we should change the way we're creating the user. Let me know what you think.

File: lockbox.go (hunk 2 above)
Comment (dsikes) [reply lockbox.go:158]: Im fine with adding a function but what advantage does that give us in this particular case?

**Pull request merged**